[2023-04-04T10:11:10.770+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.is_api_available manual__2023-04-04T10:11:09.970918+00:00 [queued]>
[2023-04-04T10:11:10.776+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.is_api_available manual__2023-04-04T10:11:09.970918+00:00 [queued]>
[2023-04-04T10:11:10.776+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-04T10:11:10.776+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-04T10:11:10.777+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-04T10:11:10.786+0000] {taskinstance.py:1309} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-04 10:11:09.970918+00:00
[2023-04-04T10:11:10.790+0000] {standard_task_runner.py:55} INFO - Started process 95835 to run task
[2023-04-04T10:11:10.792+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-04-04T10:11:09.970918+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp6_29p11v']
[2023-04-04T10:11:10.793+0000] {standard_task_runner.py:83} INFO - Job 29: Subtask is_api_available
[2023-04-04T10:11:10.840+0000] {task_command.py:389} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-04-04T10:11:09.970918+00:00 [running]> on host 1f05588790e0
[2023-04-04T10:11:10.891+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-04T10:11:09.970918+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-04T10:11:09.970918+00:00
[2023-04-04T10:11:10.892+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:11:10.899+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:11:11.560+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:11:11.560+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:12:11.618+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:12:11.624+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:12:12.101+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:12:12.101+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:13:12.162+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:13:12.168+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:13:12.666+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:13:12.666+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:14:12.722+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:14:12.728+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:14:13.178+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:14:13.178+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:15:13.222+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:15:13.228+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:15:13.669+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:15:13.669+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:16:13.704+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:16:13.709+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:16:14.165+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:16:14.165+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:17:14.227+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:17:14.233+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:17:14.727+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:17:14.727+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:18:14.786+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:18:14.791+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:18:15.254+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:18:15.254+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:19:15.267+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:19:15.272+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:19:15.726+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:19:15.727+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:20:15.789+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:20:15.796+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:20:16.283+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:20:16.283+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:21:16.344+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:21:16.348+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:21:17.621+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:21:17.622+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:22:17.682+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:22:17.687+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:22:18.227+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:22:18.227+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:23:18.289+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:23:18.295+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:23:18.995+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:23:18.995+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:24:19.055+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:24:19.061+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:24:19.495+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:24:19.496+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:25:19.558+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:25:19.563+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:25:20.052+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:25:20.052+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:26:20.110+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:26:20.117+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:26:20.609+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:26:20.609+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:27:20.706+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:27:20.714+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:27:21.173+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:27:21.174+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:28:21.202+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:28:21.208+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:28:21.698+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:28:21.698+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:29:21.758+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:29:21.763+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:29:22.265+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:29:22.265+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:30:22.302+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:30:22.310+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:30:22.760+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:30:22.760+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:31:22.820+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:31:22.828+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:31:23.296+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:31:23.296+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:32:23.358+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:32:23.367+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:32:23.841+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:32:23.842+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:33:23.902+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:33:23.908+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:33:24.385+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:33:24.385+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:34:24.446+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:34:24.451+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:34:24.901+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:34:24.901+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:35:24.963+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:35:24.968+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:35:25.423+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:35:25.423+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:36:25.482+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:36:25.487+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:36:26.055+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:36:26.055+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:37:26.117+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:37:26.122+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:37:26.585+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:37:26.586+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:38:26.647+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:38:26.653+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:38:27.141+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:38:27.141+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:39:27.202+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:39:27.208+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:39:27.671+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:39:27.671+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:40:27.732+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:40:27.738+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:40:28.221+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:40:28.221+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:41:28.282+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:41:28.288+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:41:28.800+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:41:28.800+0000] {http.py:171} ERROR - Not Found
[2023-04-04T10:42:28.858+0000] {http.py:122} INFO - Poking: https://randomuser.me/
[2023-04-04T10:42:28.863+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-04-04T10:42:29.354+0000] {http.py:170} ERROR - HTTP error: Not Found
[2023-04-04T10:42:29.354+0000] {http.py:171} ERROR - Not Found
